src/frontend/web_application/test/functional/features/24-scroll-spec.js

```diff
@@ -0,0 +1,38 @@
+const userUtil = require('../utils/user-util');
+const { home } = require('../utils/navigation');
+const { filter } = require('../utils/timeline');
+
+describe('Scroll on Timeline and Discussion', () => {
+  const EC = protractor.ExpectedConditions;
+
+  beforeAll(async () => {
+    await userUtil.signin();
+  });
+
+  beforeEach(async () => {
+    await home();
+  });
+
+  it('Reaches targets and goes back home', async () => {
+    await filter('All');
+    await browser.wait(EC.presenceOf($('.s-timeline .s-message-item')), 5 * 1000);
+    await element(by.cssContainingText(
+      '.s-timeline .s-message-item .s-message-item__title .s-message-item__excerpt',
+      'Rien du tout !'
+    )).click();
+    await browser.wait(EC.presenceOf($('.m-message')), 5 * 1000);
+    const scrollYD = await browser.executeScript(() => window.scrollY);
+    console.log(scrollYD);
+    expect(scrollYD > 0);
```

Comment:
I didn't know you can do that !

but ... does this has a better visual aspect on logs ? `expect(scrollYD).toBeGreaterThan(0)`